test/opt/test_def_use.cpp

```diff
@@ -1414,4 +1414,41 @@ INSTANTIATE_TEST_CASE_P(
       }));
 // clang-format on
 
+TEST(DefUseManagerSingleTest, GetAnnotations) {
+  // Code
+  const char* code =
+      "OpDecorate %1 Block "
+      "OpDecorate %1 RelaxedPrecision "
```

Comment:
One debug instruction is added in 92d6fd7. The debug instruction won't be counted as an annotation instruction.